rfqmsg/reject.go

```diff
@@ -102,9 +112,18 @@ var (
 	}
 )
 
+const (
+	// latestRejectVersion is the latest supported reject wire message data
+	// field version.
+	latestRejectVersion = V0
```

Comment:
nit: shouldn't this just be "lastKnownVersion"? Since we won't reject V0.

Reply:
Or just `latestVersion` similar to the name used in the next commit.

Reply:
Here I attempted to use "latest version" constants which were specific to each message type. They are all in the same package so I think they need unique names.

Reply:
Oh, I see... The word "reject" refers to the message type, not that we reject this version... Okay, never mind.